internal/config/config.go

```diff
@@ -35,6 +36,13 @@ type Configuration struct {
 	// Enduro and preservation processing.
 	SharedPath string
 
+	// CheckDuplicates enables or disables a check for SIPs that have already
+	// been processed. When enabled, the persistence configuration below will
+	// be required, and a SIP that has already been processed will fail the
+	// preprocessing workflow.
+	CheckDuplicates bool
+	Persistence     persistence.Config
```

Comment:
I think there should be a blank line after `CheckDuplicates` and no blank line after `Persistence`.